### ISSUE TITLE
Adding IPv6 Internet Node on Android has wrong keyboard

### ISSUE THREAD

**Issue by MathJud** [qaul/qaul.net#584]
When trying to add a new IPv6 Internet node on Android, one cannot enter the address, as the keyboard only allows numbers.

The keyboard must allow letters and numbers in order to enter an IPv6 address correctly.
